video_detection/video_rti.py

```diff
@@ -31,7 +32,7 @@
         infection_text = ""
 
         # check if all demographic features has been found
-        if age is not None and gender is not None and ethnicity is not None:
```

Comment:
In this case I need the check because if the classifier returns 0 it will be evaluated as `false` instead of an `int`